### PR TITLE
Anya/3291 fix hearing times

### PR DESCRIPTION
**Pull request by aroltsch**
1. Parse the hearing dates correctly
2. Travel board and video records are scheduled for that RO's timezone.
3. Travel Board master records all have start time of midnight 

Steps to Validate:
1. Navigate to the upcoming dockets page
2. Ensure that the start times are accurate.
* CO should always show in EST time
* Video should be in EST but reflect the accurate start time for that RO location
* Travel board should be in EST but reflect the accurate start time for that RO location
3. Travel board master records should always start at midnight.
4. If possible compare with Vacols UI results for the start times. 

Tested against prod copy records.

image

image

### CODE REVIEW

File: app/helpers/vacols_helper.rb [hunk 1]
```diff
@@ -8,6 +8,22 @@ def self.local_time_with_utc_timezone
     Time.utc(value.year, value.month, value.day, value.hour, value.min, value.sec)
   end
 
+  # dates in VACOLS are incorrectly recorded as UTC.
+  def self.normalize_vacols_datetime(datetime)
```
Comment (aroltsch) [inline app/helpers/vacols_helper.rb]: moved this here from the AppealRepository

File: app/mappers/hearing_mapper.rb [hunk 2]
```diff
@@ -33,11 +33,14 @@ def bfha_vacols_code(hearing_record)
       end
     end
 
-    # The hearing datetime reflect the timezone of the local RO,
+    # The TB hearing datetime reflect the timezone of the local RO,
```
Comment (aroltsch) [inline app/mappers/hearing_mapper.rb]: I re-read Jed's email and he meant that only TB records are recorded in local RO time

File: app/models/concerns/regional_office_concern.rb [hunk 3]
```diff
@@ -2,7 +2,8 @@ module RegionalOfficeConcern
   extend ActiveSupport::Concern
 
   def regional_office
-    { key: regional_office_key }.merge(VACOLS::RegionalOffice::CITIES[regional_office_key] || {})
+    { key: regional_office_key }.merge(VACOLS::RegionalOffice::CITIES[regional_office_key] ||
+                                       VACOLS::RegionalOffice::SATELLITE_OFFICES[regional_office_key] || {})
```
Comment (aroltsch) [inline app/models/concerns/regional_office_concern.rb]: Some TB master records reference Satellite offices

File: app/repositories/hearing_repository.rb [hunk 4]
```diff
@@ -2,6 +2,7 @@ class HearingRepository
   class << self
     # :nocov:
     def upcoming_hearings_for_judge(css_id)
+      css_id = "CASEFLOW_283"
```
Comment (sharonwarner) [inline app/repositories/hearing_repository.rb]: Accidental commit?

**Pull request merged**